filecoin-proofs/src/api/internal.rs

```diff
@@ -295,7 +298,7 @@ pub fn verify_post(
     };
 
     let compound_setup_params = compound_proof::SetupParams {
-        vanilla_params: &post_setup_params(sector_bytes as usize),
+        vanilla_params: &post_setup_params(comm_rs.len(), sector_config.sector_bytes()),
```

Comment:
Also, number of sectors, but derived from the commitments, this is a bit tricky to follow at times but I guess those are the only oracles of how many sectors we are working with at the moment the functions are called.